tests/system-tests/rdscore/internal/rdscorecommon/sriov-rootless-dpdk.go

```diff
@@ -0,0 +1,861 @@
+package rdscorecommon
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/link"
+	"gopkg.in/k8snetworkplumbingwg/multus-cni.v4/pkg/types"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/deployment"
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/apiobjectshelper"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	. "github.com/openshift-kni/eco-gotests/tests/system-tests/rdscore/internal/rdscoreinittools"
+)
+
+const (
+	serverDPDKDeploymentName = "rootless-dpdk-server"
+	dpdkServerMac            = "60:00:00:00:00:04"
+	serverPodLabel           = "rds-app=rootless-dpdk-server"
+	maxMulticastNoiseRate    = 5000
+	minimumExpectedDPDKRate  = 1000000
+
+	tapOneInterfaceName            = "ext0"
+	tapTwoInterfaceName            = "ext1"
+	tapThreeInterfaceName          = "ext2"
+	firstInterfaceBasedOnTapTwo    = "ext1.1"
+	secondInterfaceBasedOnTapTwo   = "ext1.2"
+	firstInterfaceBasedOnTapThree  = "ext2.1"
+	secondInterfaceBasedOnTapThree = "ext2.2"
+)
+
+var (
+	deploymentNamespace          = RDSCoreConfig.RootlessDPDKNamespace
+	dpdkVlanID                   = RDSCoreConfig.RootlessDPDKVlanID
+	dummyVlanID                  = RDSCoreConfig.RootlessDPDKDummyVlanID
+	dpdkDeploymentSAName         = RDSCoreConfig.RootlessDPDKDeploymentSA
+	firstInterfaceBasedOnTapOne  = fmt.Sprintf("%s.%s", tapOneInterfaceName, dpdkVlanID)
+	secondInterfaceBasedOnTapOne = fmt.Sprintf("%s.%s", tapOneInterfaceName, dummyVlanID)
+	dpdkNetworkOne               = RDSCoreConfig.RootlessDPDKNetworkOne
+	dpdkNetworkTwo               = RDSCoreConfig.RootlessDPDKNetworkTwo
+	dpdkPolicyTwo                = RDSCoreConfig.RootlessDPDKPolicyTwo
+	dpdkClientDeploymentName     = RDSCoreConfig.RootlessDPDKClientDeploymentName
+	dpdkClientVlanMac            = RDSCoreConfig.RootlessDPDKClientVlanMac
+	dpdkClientMacVlanMac         = RDSCoreConfig.RootlessDPDKClientMacVlanMac
+	dpdkClientIPVlanMac          = RDSCoreConfig.RootlessDPDKClientIPVlanMac
+	dpdkClientIPVlanIP           = RDSCoreConfig.RootlessDPDKClientIPVlanIPv4
+	dpdkClientIPVlanIPDummy      = RDSCoreConfig.RootlessDPDKClientIPVlanIPv4Dummy
+
+	rootUser         = int64(0)
+	waitTimeout      = 3 * time.Minute
+	psiWaitTimeout   = 10 * time.Second
+	psiRetryInterval = time.Second
+
+	serverDeploymentLabelMap = map[string]string{"rds-app": "rootless-dpdk-server"}
```

Comment:
shouldn't this be based on _serverPodLabel_ constant?

Reply:
yes, it is similar, but the map format